lib/FolderRow/FolderRowStory.js

```diff
@@ -0,0 +1,53 @@
+import React, { Fragment } from 'react';
```

Comment:
Feels like we should have a `stories` directory, even if it just has an `index.js` file. 

I feel like keeping the components and stories at the same level is a bit messy.

Reply:
No more `index.js` files, they make the codebase very difficult to search.

However, you could have multiple so I agree there 👍

Reply:
Good point!